pallets/nft/src/lib.rs

```diff
@@ -329,17 +329,13 @@ pub mod pallet {
 		/// # <weight>
 		/// - depends on the arguments
 		/// # </weight>
-		///
-		/// FIXME (ToZ)
-		/// The [_static_proofs] parameter seems no more used. We did not remove it, as it
-		/// may break the coupling with other (client) components.
 		#[pallet::weight(<T as Config>::WeightInfo::validate_mint())]
 		pub fn validate_mint(
 			origin: OriginFor<T>,
 			anchor_id: T::Hash,
 			deposit_address: DepositAddress,
-			proofs: Vec<Proof<T::Hash>>,
-			_static_proofs: [T::Hash; 3],
+			proofs: Vec<Proof<H256>>,
```

Comment:
not a big fan of type assumptions here. Any reason why we can use the exisiting `T::Hash` ?

Reply:
I have type mismatches otherwise, when passing static_proofs to the ProofVerifier::new function.